__tests__/topics.test.js

```diff
@@ -0,0 +1,47 @@
+const app = require("../app");
+const request = require("supertest");
+let db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("topics", () => {
+  describe("GET /api/topics", () => {
+    test("GET 200 from /api/topics", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("GET data from /api/topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body: { topics } }) => {
+          topics.forEach((topic) => {
```

Comment:
Before iterating over data, we should check the length to help the test fail faster if the length is 0 for example